### DISCUSSION TITLE
Difference to HA Conditional Card

### DISCUSSION THREAD

**Discussion by codebude** [PiotrMachowski/Home-Assistant-Lovelace-Local-Conditional-card#24] [Q&A] [answered]
How does this differ from the official conditional card? https://www.home-assistant.io/dashboards/conditional/

**Comment by PiotrMachowski** [comment, marked as answer]
Official card depends on entity states to change its visibility. This card doesn't - it runs directly in browser (if you change visibility on one device the change is not visible on other devices)
